### PR TITLE
Fix: autofill on the BaseExpensiInput

### LINKED ISSUE #4597: Web - New text input does not work with autofill
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Go to https://staging.new.expensify.com
2. Log in with expensifail account
3. Navigate to the company page https://staging.new.expensify.com/bank-account/company.
4. Observe the page

## Expected Result:
When using auto-fill, the label should not overlap the value in the text box. 

## Actual Result:
When the text input is auto-filled, the label overlaps with the value.

## Workaround:
Don't use autofill (unacceptable)

## Platform:
- Web

**Version Number:**
1.0.85-0
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos: 

Screen Shot 2021-08-12 at 2 54 56 PM

**Expensify/Expensify Issue URL:**

[View all open jobs on Upwork](https://www.upwork.com/ab/jobs/search/?q=Expensify%20React%20Native&sort=recency&user_location_match=2)

### PR DESCRIPTION
**Pull request by parasharrajat**
### Fixed Issues

$ Fixes #4597

### Tests | QA Steps
1. Open any form on NewDot where data is auto-filled for you on the Web Platform.
2. Label should be correctly placed.

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop

#### Web | Mobile Web | Desktop

image

### CODE REVIEW

**Comment by parasharrajat** [comment]
Don't forget to CP it.

**Comment by Luke9389** [comment]
testing now @parasharrajat

**Comment by Luke9389** [comment]
Nice job!
Screen Shot 2021-08-12 at 4 11 48 PM

**Comment by Luke9389** [comment]
Ignoring the Company Address field since I'm fixing that elsewhere 👍

**Review by Luke9389 — DISMISSED** [review]

File: src/components/ExpensiTextInput/BaseExpensiTextInput.js [hunk 1]
```diff
@@ -44,20 +48,36 @@ class BaseExpensiTextInput extends Component {
...
+        this.activateLabel();
+    }
+
+    onBlur() {
+        if (this.props.onBlur) { this.props.onBlur(); }
+        this.setState({isFocused: false});
+        this.deactivateLabel();
+    }
+
+    setValue(v) {
```
Comment (Jag96) [inline src/components/ExpensiTextInput/BaseExpensiTextInput.js:60]: Add [jsdoc](https://github.com/Expensify/App/blob/main/STYLE.md#jsdocs) for param type

**Review by Jag96 — CHANGES_REQUESTED** [review]
LGTM, just one request for jsdocs

File: src/components/ExpensiTextInput/BaseExpensiTextInput.js (hunk 1 above)
Comment (parasharrajat) [reply src/components/ExpensiTextInput/BaseExpensiTextInput.js]: Done

**Review by Jag96 — APPROVED** [review]

**Review by Luke9389 — APPROVED** [review]

**Comment by botify** [comment]
This has been deployed to production and is now subject to a 7-day regression period.
If no regressions arise, payment will be issued on 2021-08-24.  :confetti_ball:

**Comment by botify** [comment]
This has been deployed to production and is now subject to a 7-day regression period.
If no regressions arise, payment will be issued on 2021-09-06.  :confetti_ball:

**Pull request merged**